### PR TITLE
[WIP] Remove noao.edu URLs

### PR DESCRIPTION
**Pull request by weaverba137**
This PR removes all possible `noao.edu` URLs and replaces them with valid `noirlab.edu` URLs wherever possible. 

There are still a handful of `ftp` URLs that need to be dealt with, so don't merge yet.

In order to simplify further documentation of raw data access, I have moved all raw data documentation to a single file `pages/rawdata.rst`.

Side note: In addition to an older version of Nikola, `docutils<0.17` must be installed for the gallery pages to render correctly.

### CODE REVIEW

File: pages/dr3/files.rst
Comment (geordie666) [inline pages/dr3/files.rst:549]: My preference would be to just completely remove these "See the `raw data page`" entries (one on each of the dr1-dr9 files pages). I think it's a good idea to have this in a generic drop-down menu instead, and I think it would be a clean break to simply not mention the raw data page outside of that menu.

I suppose there's a backward-compatibility argument for leaving links in place, though, so, it's your call.

**Review by geordie666 — APPROVED** [review]
I rendered and served the web pages in my local browser and everything looks good.

I included one minor suggestion, but it's just a matter of taste. Feel free to merge this when you're ready without following that suggestion.

File: CHANGES.md
Comment (geordie666) [inline CHANGES.md:5]: Don't forget to document the changes for the raw data page, too.

File: pages/dr3/files.rst
Comment (weaverba137) [reply pages/dr3/files.rst:549]: Yes, it's already being added to the 'Other Data' drop down menu.  I think I'll leave the existing stubs in place though, so at least people won't wonder why the section disappeared entirely.

File: CHANGES.md
Comment (weaverba137) [reply CHANGES.md]: Done.

**Comment by weaverba137** [comment]
@stephjuneau , I have heard that you may have enough expertise in the new NOIRLab archive to be able to rebuild the raw and CP-reduced data access instructions.  Could you please help with this?

**Comment by weaverba137** [comment]
@stephjuneau, could you please take a look at the request above?

**Comment by weaverba137** [comment]
@stephjuneau, The brick coadds are available through other services, and at multiple locations. Let's focus on accessing raw and CP-reduced data, which are *only* provided by the NOIRLab archive.  Though if those are accessible via SIA, that works too.

Basically, we currently have *zero* working instructions on how to access raw and CP-reduced data, and I would like to at least have *one*, and we don't care if it's NOIRLab archive or Data Lab SIA.

**Comment by arjundey** [comment]
On this page: 
	https://www.legacysurvey.org/dr9/catalogs/#galactic-extinction-coefficients
we list the A/E(B-V) for [u,g,r,i,z,Y] but do not actually state that these are the filters! Could we please change this sentence:

"These coefficients are A / E(B-V) = 3.995, 3.214, 2.165, 1.592, 1.211, 1.064 (note that these are slightly different from the coefficients in [Schlafly & Finkbeiner 2011](https://ui.adsabs.harvard.edu/abs/2011ApJ...737..103S/abstract))."
 
to  

"These coefficients are A / E(B-V) = 3.995, 3.214, 2.165, 1.592, 1.211, 1.064 for the DECam u, g, r, i, z, Y filters, respectively. Note that these are slightly different from the coefficients in [Schlafly & Finkbeiner 2011](https://ui.adsabs.harvard.edu/abs/2011ApJ...737..103S/abstract).”

**Comment by weaverba137** [comment]
@arjundey, can do, but does that mean that has to be changed for every data release?  Presumably we want the information to be accurate for DR8, DR7, DR6,...

**Comment by arjundey** [comment]
> @arjundey, can do, but does that mean that has to be changed for every data release? Presumably we want the information to be accurate for DR8, DR7, DR6,...

Hi Ben - that would be nice, but non-critical. I assume people just "figured it out" in the past, but i just did not want this to be extra work going forward. Thanks!

**Comment by weaverba137** [comment]
@arjundey, I've made the update.  It was easy to copy-paste the change into the other data releases.

**Comment by geordie666** [comment]
@weaverba137: Could you add one small update to the pages, please. Apologies that I keep pushing this your way, but that's the perils of an open PR I guess!

On the dr9 files page:

https://github.com/legacysurvey/legacysurvey/blob/main/pages/dr9/files.rst#ccds-annotated-camera-dr9-fits-gz

Can you update:
```
Mean sky counts level per second per pixel (AVSKY divided by EXPTIME) in the CP-processed frames measured (with iterative rejection) for each CCD in the image section [500:1500,1500:2500].
```
to include an additional couple of sentences:
```
Mean sky counts level per second per pixel (AVSKY divided by EXPTIME) in the CP-processed frames measured (with iterative rejection) for each CCD in the image section [500:1500,1500:2500]. DECam exposure data is in electrons. Mosaic and 90prime are in electrons/sec. Sky counts are normalized to maintain a mean level from the original gain-corrected ADU.
```

**Comment by weaverba137** [comment]
@geordie666, that sentence appears to describe the column ``ccdskycounts`` in `survey-ccds-<camera>-dr9.fits.gz`, not `ccds-annotated-<camera>-dr9.fits.gz`.  Did you actually mean `survey-ccds-...`?

**Comment by geordie666** [comment]
Ben: Actually, maybe we should change the `ccdskycounts` description to remove the first "per second" so as to avoid confusion with the second "/sec". The result would be:

Mean sky counts level per pixel (AVSKY divided by EXPTIME) in the CP-processed frames measured (with iterative rejection) for each CCD in the image section [500:1500,1500:2500]. DECam exposure data is in electrons. Mosaic and 90prime are in electrons/sec. Sky counts are normalized to maintain a mean level from the original gain-corrected ADU.

Thanks again.

**Comment by weaverba137** [comment]
OK, change is committed, again.  Please review.

**Comment by geordie666** [comment]
Looks good to me, thanks.

**Comment by weaverba137** [comment]
Due to the uncertainty about the amount of work involved in #143, we're going to go ahead and merge this PR without addressing that issue.

**Pull request merged**
